frontend/src/components/ImageLikeUI/LikeUI.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import Pic from "./justaPhoto.avif";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHeart as fasHeart } from "@fortawesome/free-solid-svg-icons";
+import { faHeart as farHeart } from "@fortawesome/free-regular-svg-icons";
+
+import "./_ImageLikeUi.css";
+function LikeUI() {
+  const [solid, setSolid] = useState(false);
+  window.addEventListener("dblclick", () => {
```

Comment:
there is no need to add event listener to the window object. Please remove this. It should be an event on each image.